Source/Krypton Toolkit/Directory.Build.props

```diff
@@ -1,156 +1,71 @@
 <Project>
-
 	<Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../../'))" />
-
+	
+<!--Handle AssemblyInfo-->
 	<PropertyGroup>
+		<GenerateAssemblyInfo>true</GenerateAssemblyInfo>
 		<NeutralLanguage>en</NeutralLanguage>
-		<Nullable>enable</Nullable>
+		<Authors>Peter Wagner (A.K.A Wagnerp) and Simon Coghlan (A.K.A Smurf-IV), et al.</Authors>
+		<Copyright>© Component Factory Pty Ltd, 2006 - 2016. Then modifications by Peter Wagner (aka Wagnerp) and Simon Coghlan (aka Smurf-IV) 2017 - 2022. All rights reserved.</Copyright>
```

Comment:
Can the order of these be swapped to reflect,who has been working n it most recently..

Reply:
Yes, it was just a left over from the standard toolkit files.